docs/architecture/raft.rst

```diff
@@ -30,31 +34,34 @@ Note that when you have a two-DC cluster with the same number of nodes in each D
 of the DCs is down.
 **We recommend configuring three DCs per cluster to ensure that the cluster remains available and operational when one DC is down.**
 
-.. _enabling-raft-existing-cluster:
+.. _raft-schema-changes:
+
+Safe Schema Changes with Raft
+-------------------------------
+In ScyllaDB, schema is based on :doc:`Data Definition Language (DDL) </cql/ddl>`. In earlier ScyllaDB versions, schema changes were tracked via the gossip protocol, which might lead to schema conflicts if the updates are happening concurrently.
 
-Enabling Raft
----------------
+Implementing Raft eliminates schema conflicts and allows full automation of DDL changes under any conditions, as long as a quorum
+of nodes in the cluster is available. The following examples illustrate how Raft provides the solution to problems with schema changes.
+
+* A network partition may lead to a split-brain case, where each subset of nodes has a different version of the schema.
 
-ScyllaDB Open Source 5.2 and later, and ScyllaDB Enterprise 2023.1 and later come equipped with a procedure that can setup Raft-based consistent cluster management in an existing cluster. We refer to this as the **Raft upgrade procedure** (do not confuse with the :doc:`ScyllaDB version upgrade procedure </upgrade/index/>`).
+     With Raft, after a network split, the majority of the cluster can continue performing schema changes, while the minority needs to wait until it can rejoin the majority. Data manipulation statements on the minority can continue unaffected, provided the :ref:`quorum requirement <raft-quorum-requirement>` is satisfied.
 
-.. warning::
-    In ScyllaDB Open Source 5.5 and ScyllaDB Enterprise 2024.2 Raft is mandatory.
+* Two or more conflicting schema updates are happening at the same time. For example, two different columns with the same definition are simultaneously added to the cluster. There is no effective way to resolve the conflict - the cluster will employ the schema with the most recent timestamp, but changes related to the shadowed table will be lost.
 
-When all the nodes in the cluster are upgraded to ScyllaDB Open Source 5.5 or ScyllaDB Enterprise 2024.2, the cluster will start the **Raft upgrade procedure**.
+     With Raft, concurrent schema changes are safe.
 
-.. only:: opensource
 
-    See :doc:`the upgrade guide from 5.4 to 5.5 </upgrade/index>` for details.
 
-.. warning::
-    Once enabled, Raft cannot be disabled on your cluster.
+In summary, Raft makes schema changes safe, but it requires that a quorum of nodes in the cluster is available.
 
 .. _verify-raft-procedure:
 
 Verifying that the Raft upgrade procedure finished successfully
 ========================================================================
 
-.. scylladb_include_flag:: note-enabling-consistent-topology-changes.rst
+You must perform the following procedure only once - when the Raft-based 
+schema changes feature has been enabled in your cluster for the first time.
```

Comment:
You must also perform it after going through the manual Raft recovery procedure (https://opensource.docs.scylladb.com/master/troubleshooting/handling-node-failures.html#manual-recovery-procedure)

**every time** one does the manual Raft recovery procedure (hopefully never, because it's for disaster scenarios -- loss of majority), one has to do this validation.

Perhaps we can preamble the statement with "Normally" or something. "Normally, you must perform ..."